### ISSUE TITLE
Include nuspec in the repsitory

### ISSUE THREAD

**Issue by skeetor** [google/googletest#2094]
I tried to use the official gtest package, but for some reason it doesn't work in my VS 2017 (I get all kind of compiler problems).
I wrote my own gtest.nuspec and gtest.target so I can easily include gtest in my projects. I wonder though why the nuspec /targets is not included in the repository to make it easier when build from source?

If it is ok, I can create a PR and add my files.

**Comment by gennadiycivil** [comment]
googletest itself is not a right place to include specific support for (NuGet I am guessing? it is not really clear from the description)

**Comment by Aralox** [comment]
Hi @gennadiycivil , I really do believe we should include the `.nuspec` and all related nuget files in this repo. I see it as sort of the same as including CMake files - it makes it much easier for people to get up and running with the project. 

It looks like this is the most 'official' package: https://www.nuget.org/packages/gmock/ as it says it's by Google, the releases follow the releases here and the project page links to this repo. But from your comments on other issues, it looks like that package isn't actually 'official'? I tried contacting 'stailgot' through the nuget site to suggest a change to one of the files in the nuget package (`gmock.targets`), but have not received a response, and am wondering if my message was swallowed into the nuget.org email system somewhere. I have included the change I suggested below if you are interested.

Anyway, I really believe we should include all nuget-related files in this repo, and start publishing an official official package (or make it clear that the one I linked 'gmock' is the official one). They are very small and straightforward, as the package wouldn't include any built binaries but instead just have the source and target file to compile against. It would remove all ambiguity between this repo and the current various nuget packages for google test/mock, and would definitely encourage many people to start using google test (many people in my team and company too)!

Thanks,
Pravin

```
diff --git a/DVM_DEVELOP/gmock.targets b/DVM_DEVELOP/gmock.targets
index 04303f7e..2b163222 100644
--- a/DVM_DEVELOP/gmock.targets
+++ b/DVM_DEVELOP/gmock.targets
@@ -11,12 +11,14 @@
   <ItemGroup>
     <ClCompile Include="$(MSBuildThisFileDirectory)..\..\lib\native\src\gtest\gtest-all.cc">
       <PreprocessorDefinitions>;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <PrecompiledHeader>NotUsing</PrecompiledHeader>
     </ClCompile>
   </ItemGroup>
 
   <ItemGroup>
     <ClCompile Include="$(MSBuildThisFileDirectory)..\..\lib\native\src\gmock\gmock-all.cc">
       <PreprocessorDefinitions>;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <PrecompiledHeader>NotUsing</PrecompiledHeader>
     </ClCompile>
   </ItemGroup>
 </Project>
\ No newline at end of file

```
